bin/edge_cases/knapsack_pro_queue_rspec_in_background

```diff
@@ -10,11 +10,21 @@ export KNAPSACK_PRO_CI_NODE_TOTAL=2
 export KNAPSACK_PRO_BRANCH=test-1
 export KNAPSACK_PRO_COMMIT_HASH=commit-${1:-1}
 #export KNAPSACK_PRO_TEST_FILE_PATTERN="spec/controllers/**/*_spec.rb"
-#export KNAPSACK_PRO_RSPEC_SPLIT_BY_TEST_EXAMPLES=true
+#export KNAPSACK_PRO_TEST_FILE_PATTERN="spec/controllers/articles_controller_spec.rb"
+export KNAPSACK_PRO_RSPEC_SPLIT_BY_TEST_EXAMPLES=true
+#export KNAPSACK_PRO_SLOW_TEST_FILE_PATTERN="{spec/slow_shared_examples_spec.rb,spec/features/calculator_spec.rb}"
+#export KNAPSACK_PRO_SLOW_TEST_FILE_PATTERN="{spec/features/calculator_spec.rb}"
+export KNAPSACK_PRO_SLOW_TEST_FILE_PATTERN="{spec/timecop_spec.rb}"
 #export KNAPSACK_PRO_LOG_LEVEL=warn
 
 KNAPSACK_PRO_CI_NODE_INDEX=0 \
-  bundle exec rake "knapsack_pro:queue:rspec[-f d]" &
+  bundle exec rake "knapsack_pro:queue:rspec[-f d]" #&
 
 KNAPSACK_PRO_CI_NODE_INDEX=1 \
```

Comment:
@shadre you can comment out this to run tests only on one of nodes.

the last char `&` makes tests run in background so if both commands for node index 0 and 1 have `&` at the end then they will run in parallel.